testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -47,6 +52,7 @@ pub struct PerformanceBenchmark {
     percent_nodes_down: usize,
     duration: Duration,
     trace: bool,
+    tps: u64,
```

Comment:
This field should be option too, by same logic?)

Reply:
I think this filed is constructed by PerformanceBenchmarkParams, we set it is optional in PerformanceBenchmarkParams and we may use 0 to be the default. I can change it to option as well

Reply:
Yeah, this should be option for the same reason - avoiding 'magical' values